app/views/companies/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= @company.symbol %><br>
```

Comment:
Maybe you could add some styling here, for example:

```suggestion
<h2><%= @company.symbol %></h2>
```

Reply:
Yes of course, but this is only temporary, anyway it looks better even temp :)